src/screens/DiscoveryScreenV2/elements/DiscoveryTitleSeparator.js

```diff
@@ -1,36 +1,31 @@
 import * as React from 'react';
-import {Text} from 'react-native';
+import { Text } from 'react-native';
 
-import {fonts} from '../../../utils/fonts'
-import {COLORS} from '../../../utils/theme';
+import { colors } from "../../../utils/colors"
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for colors has been updated. Ensure that the `colors` module exports the required color constants and that they are correctly referenced throughout the codebase.
```diff
- import { COLORS } from "../../../utils/theme"
+ import { colors } from "../../../utils/colors"
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->